### PR TITLE
feat(payment): PAYPAL-983 updated paypal documentation

### PR DESCRIPTION
**Pull request by andriiVitvitskyi1990**
## What?
Updated paypal documentation

## Why?
According to task PAYPAL-983
## Testing / Proof
...

@bigcommerce/checkout @bigcommerce/payments

### CODE REVIEW

**Review by vneutrino — APPROVED** [review]
LGTM

File: src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts [hunk 1]
```diff
@@ -77,18 +77,23 @@ export type PaypalCommerceInitializeOptions = PaypalCommercePaymentInitializeOpt
...
      * The CSS selector of a container where the payment widget should be inserted into.
+     * It's necessary to specify this parameter when using Paypal.
+     * Without it Paypal will not work.
+     * For example if you have element with id="paypal-container" - then your container will look like "#paypal-container"
      */
     container: string;
 
     /**
      *  The Client ID of the Paypal App
+     *  You wont be able to initialize Paypal without Client ID
```
Comment (icatalina) [inline src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts:88]: I'm not sure this is appropriate. Anything that doesn't have a `?:` is required. We could make it more apparent when we generate the documentation, but I don't think we should be adding redundant text all across the codebase.

**Review by icatalina — CHANGES_REQUESTED** [review]

**Comment by animesh1987** [comment]
@andriiVitvitskyi1990 is this PR still valid? If not can we please close this?

File: src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts (hunk 1 above)
Comment (andriiVitvitskyi1990) [reply src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts:88]: Fixed

File: src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts [hunk 2]
```diff
@@ -89,6 +90,7 @@ export interface PaypalCommercePaymentInitializeOptions {
      * The CSS selector of a container where the alternative payment methods fields widget should be inserted into.
      * It's necessary to specify this parameter when using Alternative Payment Methods.
```
Comment (animesh1987) [inline src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts:91]: Is there a better way to surface this, as @icatalina pointed previously that anything with `?:` is not required 🤷‍♂️

Comment (andriiVitvitskyi1990) [reply src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts:91]: As i see container is required and don't have ? (optional sign)

**Review by animesh1987 — APPROVED** [review]

File: src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts (hunk 2 above)
Comment (icatalina) [reply src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts]: Hey @andriiVitvitskyi1990, you are right, `container` is already flagged as mandatory. The goal is to use a consistent way of flagging this across the codebase, as opposed to adding custom notes to every mandatory field. 

If we feel the documentation is not clear enough using `?:` and the `Optional` label (https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/paypalcommercepaymentinitializeoptions.md#optional-apmfieldscontainer) we can look into ways of improving it by tweaking the markdown generator.

Comment (animesh1987) [reply src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts]: @icatalina  We addressed this offline, @andriiVitvitskyi1990 thought the comment was for `container` field 😄 And then adjusted the text accordingly.

File: src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts [hunk 3]
```diff
@@ -87,8 +88,8 @@ export interface PaypalCommercePaymentInitializeOptions {
 
     /**
      * The CSS selector of a container where the alternative payment methods fields widget should be inserted into.
-     * It's necessary to specify this parameter when using Alternative Payment Methods.
-     * Without it alternative payment methods will not work.
+     * If you want  Alternative Payment Methods to be displayed - you need to specify apmFieldsContainer. Without it alternative payment methods will not work.
```
Comment (icatalina) [inline src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts:91]:
```suggestion
     * If you want _Alternative Payment Methods_ to be displayed you need to specify `apmFieldsContainer`. Without it, alternative payment methods will not work.
```

File: src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts [hunk 4]
```diff
@@ -87,8 +88,8 @@ export interface PaypalCommercePaymentInitializeOptions {
...
+     * For example if you have element with id="apm-fields-container" - then your container will look like "#apm-fields-container"
```
Comment (icatalina) [inline src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts:92]:
```suggestion
     * For example if you have an element with `id="apm-fields-container"`, then your container will look like `#apm-fields-container`
```

**Review by icatalina — APPROVED** [review]
Happy with the text, just added a few comments for the format.

**Pull request closed without merging**